src/patternfly/demos/Page/examples/Page.md

```diff
@@ -17,6 +17,58 @@ Using the `.pf-m-full-height` modifier class on the page component eliminates th
 {{> page-template page-template--id="page-demo-full-height" page-template--modifier="pf-m-full-height"}}
 ```
 
+### Using flex layout instead of multiple body elements
+```hbs isFullscreen
+{{> page-template page-template--id="multiple-body-elements-demo"}}
+
+{{#*inline "page-template-sidebar"}}
+  {{#> page-sidebar page-sidebar--ExcludeSidebarBody="true"}}
+    {{#> page-sidebar-body page-sidebar-body--modifier="pf-m-fill"}}
+      {{#> l-flex l-flex--modifier="pf-m-column pf-u-h-100"}}
+        {{#> l-flex-item}}
+          {{> page-template-sidebar-nav}}
+        {{/l-flex-item}}
+        {{#> l-flex-item l-flex-item--modifier="pf-u-px-md pf-u-px-lg-on-xl"}}
+          other content
+        {{/l-flex-item}}
+        {{#> l-flex-item l-flex-item--modifier="pf-u-px-md pf-u-px-lg-on-xl pf-u-mt-auto"}}
+          footer content
+        {{/l-flex-item}}
+      {{/l-flex}}
+    {{/page-sidebar-body}}
+  {{/page-sidebar}}
+{{/inline}}
+```
+
+### Multiple sidebar body elements
+```hbs isFullscreen
+{{> page-template page-template--id="multiple-body-elements-demo"}}
+
+{{#*inline "page-template-sidebar"}}
+  {{#> page-sidebar page-sidebar--ExcludeSidebarBody="true" page-sidebar--modifier="pf-m-expanded"}}
+    {{#> page-sidebar-body page-sidebar-body--modifier="pf-m-page-insets"}}
+      {{#> content}}
+        <p>Custom sidebar content</p>
+      {{/content}}
+    {{/page-sidebar-body}}
+    {{> divider}}
+    {{#> page-sidebar-body}}
+      {{> page-template-sidebar-nav}}
+    {{/page-sidebar-body}}
+    {{#> page-sidebar-body page-sidebar-body--modifier="pf-m-page-insets"}}
+      {{#> content}}
+        <p>Custom sidebar content</p>
+      {{/content}}
+    {{/page-sidebar-body}}
+    {{#> page-sidebar-body page-sidebar-body--modifier="pf-m-align-bottom pf-m-page-insets"}}
```

Comment:
I'm still not sure why we need an `align-bottom` class if body elements can fill, and you can put whatever you want in a body element. Also I wonder if these elements should act like page sections - the only/last child will fill, and we have fill/no-fill support - that's how you'd do a footer in the page main content area, which is already pretty familiar using the page and cards.